requirements.txt

```diff
@@ -1,3 +1,3 @@
-asyncpg==0.18.3
-SQLAlchemy>=1.2
-aiocontextvars==0.2.1
+asyncpg~=0.18
+SQLAlchemy>=1.2,<1.3
```

Comment:
No objection, but I guess we can add version check now. something like if SQLAlchemy.version >= 1.3, use the other method.

Reply:
Yeah, I'm thinking about the same.